Sources/Real/AlgebraicField.swift

```diff
@@ -0,0 +1,85 @@
+//===--- AlgebraicField.swift ---------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift Numerics open source project
+//
+// Copyright (c) 2019 Apple Inc. and the Swift Numerics project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+/// A type modeling an algebraic [field]. Refines the `Numeric` protocol,
+/// adding division.
+///
+/// A field is a set on which addition, subtraction, multiplication, and
+/// division are defined, and behave basically like those operations on
```

Comment:
```suggestion
/// division are defined, and behaves basically like those operations on
```

Reply:
"Behave" is correct here; the subject is "addition, subtraction, ... and division".